README.md

```diff
@@ -6,10 +6,11 @@ IPTV proxy for Plex Live written in Golang
 
 1) Go to the releases page and download the correct version for your Operating System
 2) Have the .m3u file on hand from your IPTV provider of choice
-3) Run `telly` with the `-playlist` commandline argument pointing to your .m3u file. (This can be a local file or a URL) For example: `./telly -playlist=/home/github/myiptv.m3u`
-4) If you would like `telly` to attempt to the filter the m3u a bit, add the `-filterregex` commandline option. If you would like UK only tv, run `telly` with the `-uktv` commandline option. If you would like to use your own regex, run `telly` with `-useregex <regex>`, for example `-useregex .*UK.*`
-5) If `telly` tells you `[telly] [info] listening on ...` - great! Your .m3u file was successfully parsed and `telly` is running. Check below for how to add it into Plex.
-6) If `telly` fails to run, check the error. More than likely it's the formatting of the .m3u file. Common problems can be the format of the run time. Open your .m3u file in your favourite text editor and check lines starting with `#EXTINF:-1` or `#EXTINF:0`. They should be `#EXTINF:-1,` or `#EXTINF:0,` - the comma is the more important part. You can run a simple `sed` command to fix this, something like `sed -i 's/#EXTINF:-1/#EXTINF:-1,/g' myiptv.m3u` or `sed -i '' 's/#EXTINF:-1/#EXTINF:-1,/g' myiptv.m3u` if you are on macOS. If all else fails, open an issue and I'll be more than happy to help you out.
+3) Run `telly` with the `-playlist` commandline argument pointing to your .m3u file. (This can be a local file or a URL) For example: `./telly -playlist=/home/github/myiptv.m3u`  
+4) If you would like multiple `streams`/tuners use the `-streams` commandline option. Default is 1
```

Comment:
putting `-streams` in back ticks is perfectly fine, but I don't think it's needed for the `streams`/tuners bit 👍

Reply:
Done